stl/inc/__msvc_chrono.hpp

```diff
@@ -688,9 +708,11 @@ namespace chrono {
                 const long long _Part  = (_Ctr % _Freq) * period::den / _Freq;
                 return time_point(duration(_Whole + _Part));
             }
+            // clang-format on
         }
     };
-
+#undef _LIKELY_ARM
+#undef _LIKELY_X86
     _EXPORT_STD using high_resolution_clock = steady_clock;

```

Comment:
Nitpick: The macros were defined right before `now()` so for symmetry they should be undefined right after. This will also preserve the newline between `steady_clock` and `high_resolution_clock`.